dask/base.py

```diff
@@ -1195,9 +1195,13 @@ def normalize_period_dtype(dtype):
         return normalize_token(dtype.name)
 
     @normalize_token.register(type(pd.NA))
-    def normalize_na(dtype):
+    def normalize_na(na):
```

Comment:
Was this a problem before, or are you just aiming for better clarity?

Reply:
better clarity